data/channels.csv

```diff
@@ -20616,180 +20616,180 @@ NDTVRecordTVCriciuma.br,NDTV Record TV Criciuma,NDTV Record TV Criciúma,Rede Re
-NETVToronto.ca,NETV Toronto,,,,CA,,,c/CA,ell,movies,FALSE,,,,,https://i.imgur.com/WXRBuOW.png
-Network10.in,Network 10,,,,IN,,,c/IN,hin,,FALSE,1964-08-01,,,https://network10.in/,https://i.imgur.com/xkWfruI.png
+NETVToronto.ca,NETV Toronto,,,,CA,CA-ON,Toronto,c/CA,ell,movies,FALSE,,,,https://www.netvtoronto.com/,https://i.imgur.com/WXRBuOW.png
+Network10.in,Network 10,,,,IN,,Delhi,c/IN,hin,general,FALSE,1964-08-01,,,https://network10.in/,https://i.imgur.com/xkWfruI.png
 NetXXL.si,Net XXL,,,,SI,,,c/SI,eng,xxx,TRUE,,,,https://tv2go.t-2.net/tv/epg/1000228/,https://i.imgur.com/jn54LMl.png
 NeuralFocused.us,Neural Focused,,,,US,,,c/US,eng,music,FALSE,,,,,https://i.imgur.com/ymFMMn8.png
-NevisTelevision.kn,Nevis Television,NTv,,,KN,,,c/KN,eng,general,FALSE,,,,https://www.nevistvonline.com/,https://i.ibb.co/gJ2y0dq/NTv-Logo-3-0.jpg
-NevoMusic.uz,NevoMusic,,,,UZ,,,c/UZ;c/RU,uzb,music,FALSE,,,,https://nevomusic.net/,https://i.ibb.co/pxBYNNS/channels4-profile-1.jpg
-NevoMusicTj.uz,NevoMusicTj,,,,UZ,,,c/TJ,tgk,music,FALSE,,,,,https://i.ibb.co/Vwnz1Mw/channels4-profile-3.jpg
-NewAnimeChannel.cn,New Anime Channel,新动漫频道,,,CN,,,c/CN,zho,,FALSE,,,,,https://i.imgur.com/QEGfqBe.png
+NevisTelevision.kn,Nevis Television,NTv,,The Department of Information,KN,,Nevis,c/KN,eng,general,FALSE,2001-01-12,,,https://www.nevistvonline.com/,https://i.ibb.co/gJ2y0dq/NTv-Logo-3-0.jpg
+NevoMusic.uz,NevoMusic,,,,UZ,,Tashkent,c/UZ;c/RU,uzb,music,FALSE,,,,https://nevomusic.net/,https://i.ibb.co/pxBYNNS/channels4-profile-1.jpg
+NevoMusicTj.uz,NevoMusicTj,,,,UZ,,Tashkent,c/TJ,tgk,music,FALSE,,,,,https://i.ibb.co/Vwnz1Mw/channels4-profile-3.jpg
+NewAnimeChannel.cn,New Anime Channel,新动漫频道,,,CN,,,c/CN,zho,animation,FALSE,,,,,https://i.imgur.com/QEGfqBe.png
 NewaTVHD.np,Newa TV HD,,,,NP,,Kathmandu,c/NP,nep,general,FALSE,,,,,https://i.imgur.com/Wde7NcJ.jpg
 NewBedfordEducationalAccessChannel.us,New Bedford Educational Access Channel,,,,US,US-MA,New Bedford,s/US-MA,eng,education,FALSE,,,,http://173.9.65.75/CablecastPublicSite/watch/2?channel=2,https://i.imgur.com/1vIaOLn.png
 NewBedfordGovernmentAccessChannel.us,New Bedford Government Access Channel,,,,US,US-MA,New Bedford,s/US-MA,eng,legislative,FALSE,,,,http://173.9.65.75/CablecastPublicSite/?channel=1,https://i.imgur.com/Lgn2i24.png
 NewBedfordPublicAccessChannel.us,New Bedford Public Access Channel,,,,US,US-MA,New Bedford,s/US-MA,eng,general,FALSE,,,,http://173.9.65.75/CablecastPublicSite/?channel=3,https://i.imgur.com/INXthEh.png
-NewcomerTV.ch,Newcomer TV,,,,CH,,,c/CH,deu;eng,music,FALSE,,,,https://iptv.label-tv.net/,https://i.ibb.co/t3VT2Bd/Newcomer-TV.jpg
-NewDayTV.gh,New Day TV,,,,GH,,,c/GH,eng,,FALSE,,,,,https://i.imgur.com/M8zv2B6.png
+NewcomerTV.ch,Newcomer TV,,,,CH,,Winterthur,c/CH,deu;eng,music,FALSE,,,,https://iptv.label-tv.net/,https://i.ibb.co/t3VT2Bd/Newcomer-TV.jpg
+NewDayTV.gh,New Day TV,,,,GH,,Takoradi,c/GH,eng,general,FALSE,,,,https://www.youtube.com/channel/UCUi5ZjUnQegmIOM-Z8TDxTA,https://i.imgur.com/M8zv2B6.png

```

Comment:
Once again, this field is created to store only links to the official channel website.

```suggestion
NewDayTV.gh,New Day TV,,,,GH,,Takoradi,c/GH,eng,general,FALSE,,,,,https://i.imgur.com/M8zv2B6.png
```

Reply:
Fixed.